src/views/Home.jsx

```diff
@@ -11,12 +12,14 @@ export function Home({ data, setListPath }) {
 			<AddList setListPath={setListPath} />
 			<ul>
 				{data.map((list) => (
-					<SingleList
-						key={list.name}
-						name={list.name}
-						path={list.path}
-						setListPath={setListPath}
-					/>
+					<Link key={list.id} to="/list">
```

Comment:
I think from a code organization perspective, `Link` could be rendered in `SingleList` and contained within the `li` instead of inside of the map function wrapped around `SingleList`. That way both `Link` and `SingleList` don't need keys, only `SingleList` would need `key={list.name}`. Thoughts?

Reply:
This makes sense to me!